docs/framework/data/adonet/secure-client-applications.md

```diff
@@ -33,7 +33,7 @@ Applications typically consist of many parts that must all be protected from vul
  [Windows Forms and Unmanaged Applications](../../../../docs/framework/winforms/advanced/windows-forms-and-unmanaged-applications.md)  
  Describes how to interact with unmanaged applications in a Windows Forms application.  
   
- [ClickOnce Deployment for Windows Forms Applications](https://msdn.microsoft.com/library/34d8c770-48f2-460c-8d67-4ea5684511df)  
+ [ClickOnce Deployment for Windows Forms Applications](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2008/wh45kb66(v=vs.90))  
```

Comment:
```suggestion
 [ClickOnce Deployment for Windows Forms](../../winforms/clickonce-deployment-for-windows-forms.md)  
```

Reply:
This same link is also used in another topic: make the same fix there (it doesn't need to be in this same PR): https://github.com/dotnet/docs/blob/master/docs/framework/data/adonet/sql/sql-server-express-user-instances.md